components/OgImage/EventShareInstagram.vue

```
@@ -1,148 +1,95 @@
-            text-align: center;
-            background-color: #bb0a08;
-            color: #f2f2f2;
-          "
+        <div 
+          class="info-box-title"
+          style="font-size: 52px; font-weight: 700; color: #25282b; text-align: left; line-height: 1.2;"
+        >
+          {{ title }}
+        </div>
+
+        <div 
```

ESLint

[error] Delete `·`

(prettier/prettier)